bulletroboy/cage_simulation.py

```diff
@@ -49,7 +55,7 @@ def main():
     # RUN SIM
     try:
         while True:
-
+            operator.move(FOREARM_ROLL)
             exoforce.update()
```

Comment:
given that the operator is part of the exoforce sim, for a cleaner code, I would suggest passing the movement type as a parameter with a default value to the `exoforce.update()` and apply the movement to the `self.operator` inside of the function

Reply:
Working on this. But Im out of ideas. It should be resolved with a simple line of code. But the figure doesnt react to it...

within exoforce class:
>> def update(self, case = FOREARM_ROLL)
>>           self.operator.move(case)

gives no error but doesnt move the figure. I will keep looking for a solution.

Reply:
Its on the update of ExoForceSim not ExoForce

Reply:
Done

Reply:
These movements are voluntary moments of the operator, nothing to do with the ecoforce. It should not be inside of the update function of the exoforce but in the  main simulation loop. Using exoforce.operator to access the operator though, does make sense

Reply:
Movements are back to being called from exoforce.operator.move() and separate from exoforce.update.